skgstat/DirectionalVariogram.py

```diff
@@ -452,20 +493,21 @@ def local_reference_system(self, poi):
             the original coordinates.
 
         """
-        # define a point-wise transform function
-        def _transform(p1, p2, a):
-            p = p1 - p2
-            x = p[0] * np.cos(a) - p[1] * np.sin(a)
-            y = p[0] * np.sin(a) + p[1] * np.cos(a)
-            return np.array([x, y])
+
+        assert False
```

Comment:
Is this a mistake, or is there a reason to force the updated function to fail?

Reply:
So... that function isn't used anymore, and this was just a quick way to make sure it wasn't. The whole function should be removed...